documentation/docs/components/outputs/command-outputs/nym-node-node-details.md

```diff
@@ -1,3 +1,3 @@
 ```sh
-[2m2024-11-06T12:18:46.742880Z[0m [31mERROR[0m [2mnym-node/src/config/upgrade_helpers.rs[0m[2m:[0m[2m17[0m[2m:[0m Failed to finish upgrade - failed to load config file using path '/home/m/.nym/nym-nodes/default-nym-node/config/config.toml'. detailed message: No such file or directory (os error 2)
+[2m2024-12-18T17:36:27.019221Z[0m [31mERROR[0m [2mnym-node/src/config/upgrade_helpers.rs[0m[2m:[0m[2m19[0m[2m:[0m Failed to finish upgrade - failed to load config file using path '/home/m/.nym/nym-nodes/default-nym-node/config/config.toml'. detailed message: No such file or directory (os error 2)
```

Comment:
this seems like a bad outcome

Reply:
We aren't using this right now I don't think? I need to fix this at some point though..